src/Veldrid/Vk/VkGraphicsDevice.cs

```diff
@@ -473,6 +474,12 @@ private void CreateInstance(bool debug, VulkanDeviceOptions options)
                 _surfaceExtensions.Add(CommonStrings.VK_KHR_portability_subset);
             }
 
+            if (availableInstanceExtensions.Contains(CommonStrings.VK_KHR_portability_enumeration))
+            {
+                instanceExtensions.Add(CommonStrings.VK_KHR_portability_enumeration);
+                instanceCI.flags |= VK_INSTANCE_CREATE_ENUMERATE_PORTABILITY_BIT_KHR;
```

Comment:
https://www.lunarg.com/wp-content/uploads/2022/04/Portability-Enumeration-Extension-APR2022.pdf also adds it to `ppEnabledLayerNames`.